README.md

```diff
@@ -20,6 +20,7 @@ If you want to get involved with science and research software engineering, this
  - [High Performance Computing](#high-performance-computing)
  - [Psychology](#psychology)
  - [Workflows](#workflows)
+ - [Institutions](#institutions)
```

Comment:
One nitpick - could you please put the table of contents and subsequent content in alphabetical order? So Institutions should be sandwiched between High Performance Computing and Psychology. It doesn't make a huge difference now when the list is small, but it will make it easier to find things when it gets larger.

Reply:
No problem at all. I was considering it a meta-category like "Contributing" but your suggestion makes a lot of sense!

p.s. Thanks for curating this list - great idea. Will promote.

Reply:
And thank you for your contribution! I'm off back to :zzz: :)